### PR TITLE
Add Outguess.app v1.1.1

### PR DESCRIPTION
**Pull request by Rbcafe**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by Rbcafe** [comment]
Add Outguess.app v1.1.1

File: Casks/outguess.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'outguess' do
+  version '1.1.1'
```
Comment (vitorgalvao) [inline Casks/outguess.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by Rbcafe** [comment]
outguess.rb updated

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Why did you close it? The fix was correct.

**Comment by Rbcafe** [comment]
New pull request expected ?

**Comment by vitorgalvao** [comment]
No need. You just needed to fix it so I could merge.

**Comment by Rbcafe** [comment]
New pull #22705

**Comment by jawshooah** [comment]
@Rbcafe in the future, there's no need to open a new PR when you make changes to an existing one. The PR updates with the branch, so if you make new changes they will be reflected in the original PR.

**Comment by Rbcafe** [comment]
@jawshooah  Thanks for the informations.

Best regards

**Pull request closed without merging**